VideoLocker/src/main/java/org/edx/mobile/util/InputValidationUtil.java

```diff
@@ -8,7 +8,7 @@
     public static boolean isValidEmail(String text) {
         if (text == null) 
             return false;
-        
+
         Pattern p = Pattern
                 .compile("^[_A-Za-z0-9-+]+(\\.[_A-Za-z0-9-+]+)*@[A-Za-z0-9]+(\\.[A-Za-z0-9]+)*(\\.[A-Za-z]{2,})$");
```

Comment:
This isn't part of this change, but we should probably not use this regex for validating emails, valid emails can contain unicode characters and this prohibits that.

I used this super simple regex on iOS ``".+@.+\\..+";`` to loosely approximate a valid email